tests/Resources/config/services.yml

```diff
@@ -44,3 +44,8 @@ services:
           - 'AdminLogin'
           - 'AdminQuickAccesses'
           - 'AdminTabs'
+
+  # This service is private and cannot be overriden in a test env, therefore we must set it public.
+    PrestaShop\PrestaShop\Core\Domain\Configuration\ShopConfigurationInterface:
+      class: PrestaShop\PrestaShop\Adapter\Configuration
+      public: true
```

Comment:
Does it need to be public ?

Reply:
I think there is the answer in the code comment